Source/WebKit/UIProcess/WebProcessProxy.h

```diff
@@ -143,6 +144,7 @@ class WebProcessProxy : public AuxiliaryProcessProxy, private ProcessThrottlerCl
     using WebPageProxyMap = HashMap<WebPageProxyIdentifier, WeakPtr<WebPageProxy>>;
     typedef HashMap<WebCore::FrameIdentifier, WeakPtr<WebFrameProxy>> WebFrameProxyMap;
     typedef HashMap<uint64_t, RefPtr<API::UserInitiatedAction>> UserInitiatedActionMap;
+    typedef HashMap<UUID, RefPtr<API::UserInitiatedAction>> UserInitiatedActionByAuthorizationTokenMap;
```

Comment:
Please use `using UserInitiatedActionByAuthorizationTokenMap = HashMap<UUID, RefPtr<API::UserInitiatedAction>>` in new code instead of the old-style `typedef`